generic_relations/relations.py

```diff
@@ -44,11 +44,12 @@ def field_to_native(self, obj, field_name):
         """
         value = super(GenericRelatedField, self).field_to_native(
             obj, field_name)
-        serializer = self.determine_deserializer_for_data(value)
+        if value:
```

Comment:
Ok. I was assuming that `None` was the only value we want to exit early with. If other empty values also make sense, go ahead.